src/pages/home/ReportScreen.js

```diff
@@ -321,7 +321,10 @@ class ReportScreen extends React.Component {
                             {/* Note: The report should be allowed to mount even if the initial report actions are not loaded. If we prevent rendering the report while they are loading then
                             we'll unnecessarily unmount the ReportActionsView which will clear the new marker lines initial state. */}
                             {(!this.isReportReadyForDisplay() || isLoadingInitialReportActions || isLoading) && (
-                                <ReportActionsSkeletonView containerHeight={this.state.skeletonViewContainerHeight} />
+                                <>
+                                    <ReportActionsSkeletonView containerHeight={this.state.skeletonViewContainerHeight} />
+                                    <EmojiPicker ref={EmojiPickerAction.emojiPickerRef} />
```

Comment:
I didn't understand why we should move this in this block? Also I see the linked issue is closed.

Reply:
As I mentioned in this [comment](https://github.com/Expensify/App/pull/16922#issuecomment-1496630341), I think the right solution is not to move this inside an if block, rather remove two differences.

This still isn't convincing why it wouldn't open in offline mode. Hence, I think it is better to remove linking the issue, and if required tackle it separately if we're able to reproduce it consistently.